build/openwisp_nginx/global.conf

```diff
@@ -0,0 +1,4 @@
+# Global settings across valid across all
```

Comment:
Fix wording.

Reply:
Changed to `# Custom http block configurations`